packages/icons/src/chevron-down.js

```diff
@@ -0,0 +1,11 @@
+export const ChevronDownIcon = () => (
+	<svg
+		viewBox="0 0 24 24"
+		width="24"
+		height="24"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
+		<path d="M6.50002 10L12 15L17.5 10" stroke="#CCC" strokeWidth="2" />
```

Comment:
I didn't understand what the issue is. 
In this case, I was able to change the stroke color for the selected option: https://github.com/Automattic/crowdsignal-ui/pull/264/files#diff-c55fa3ced3807e3d5d3bc37c118762305555322e7835e972648dc6c1400655daR121
And, to avoid side-effects on the other themes (which have `fill: currentColor`, I've set the `fill` property to none: https://github.com/Automattic/crowdsignal-ui/pull/264/files#diff-e25bb0ebd11548a5c85844ed0ddb254933ddc9aff257f1230b76845bfe3ac63dR236

Reply:
That's kind of what I mean... like, if you put another icon in there it might not look alright anymore. And it's kind of bad if we have to make separate CSS adjustments just based on what icon you're using - it should just be treated same as text for the most part.

Either way, this is not a blocker and more of an issue with how the SVGs are exported but I just thought I'd mention it.

Reply:
Got it! And yes, it makes sense.